### ISSUE TITLE
Error occurred on latest released coretools. This affects a customer. 

### ISSUE THREAD

**Issue by ekkizogloy** [iotile/coretools#824]
12:35 $ iotile hw --port=bled112 connect 0x53a controller sensor_graph download_stream 'output 32'
ArgumentError: get_type called on unknown type
Additional Information:
type: fw_stream
failed_external_sources: ['Entry point group: iotile.type_package, name: lib_controller_types']

**Comment by ekkizogloy** [comment]
Actually iotile-ext-cloud is required for this command to work. Can we update the error message to indicate this.

**Comment by timburke** [comment]
@ekkizogloy `iotile-ext-cloud` is not involved in this command and is not required for executing the command line that you specified above.  Can you please update with additional information about what you changed between when it didn't work and when it did?

**Comment by ekkizogloy** [comment]
I pip installed iotile-build... the command did not work.
I then pip installed iotile-ext-cloud ... then the command worked.
I then uninstalled iotile-build and iotile-ext-cloud ... then the command still worked....

My guess is that a dependency was installed which fixed the issue.

**Comment by ekkizogloy** [comment]
pip list
When Command Fails
```
12:53 $ pip list 
Package                         Version
------------------------------- -------
decorator                       4.4.0  
entrypoints                     0.3    
iotile-core                     4.1.1  
iotile-emulate                  0.4.4  
iotile-sensorgraph              1.0.6  
iotile-support-con-nrf52832-3   3.2.4  
iotile-support-firm-gpio-3      3.0.1  
iotile-support-lib-controller-3 3.7.16 
iotile-test                     1.0.2  
iotile-transport-bled112        2.0.4  
msgpack                         0.6.1  
pip                             19.1.1 
pyparsing                       2.2.2  
pyserial                        3.4    
python-dateutil                 2.8.0  
setuptools                      41.0.1 
six                             1.12.0 
sortedcontainers                2.1.0  
toposort                        1.5    
typedargs                       1.0.1  
wheel                           0.33.2 
ws4py                           0.5.1  
```

After install/uninstall of iotile-ext-cloud
Command Works
```
Package                         Version 
------------------------------- --------
certifi                         2019.3.9
chardet                         3.0.4   
decorator                       4.4.0   
entrypoints                     0.3     
future                          0.17.1  
idna                            2.8     
iotile-cloud                    0.9.9   
iotile-core                     4.1.1   
iotile-emulate                  0.4.4   
iotile-sensorgraph              1.0.6   
iotile-support-con-nrf52832-3   3.2.4   
iotile-support-firm-gpio-3      3.0.1   
iotile-support-lib-controller-3 3.7.16  
iotile-test                     1.0.2   
iotile-transport-bled112        2.0.4   
msgpack                         0.6.1   
pip                             19.1.1  
pyparsing                       2.2.2   
pyserial                        3.4     
python-dateutil                 2.8.0   
requests                        2.21.0  
setuptools                      41.0.1  
six                             1.12.0  
sortedcontainers                2.1.0   
toposort                        1.5     
typedargs                       1.0.1   
urllib3                         1.24.3  
wheel                           0.33.2  
ws4py                           0.5.1  
```

**Comment by ekkizogloy** [comment]
The issue is the 'future' package

**Comment by timburke** [comment]
@ekkizogloy Thanks for tracking this down.  It's fixable now.  I'm opening an internal issue to resolve.

**Comment by timburke** [comment]
Resolved in `iotile-support-lib-controller-3 == 3.7.17`
